### DISCUSSION TITLE
Linux Client not Syncing

### DISCUSSION THREAD

**Discussion by LarsEggimann** [nextcloud/all-in-one#3593] [Questions] [answered]
Hello

I have the nextcloud client installed on my ubuntu 22.04 and connected to my nextcloud. I had the `Too many open files` error, upon which I adjusted max number of files according to [this guide](https://www.tutorialspoint.com/fixing-the-too-many-open-files-error-in-linux). This worked as I did no longer revieve the error. Yet for some reason the sync will not finish no matter how long i wait (waited over 12 hours) and it just stops saying 0s remaining and not uploading any more files to my cloud.

Is this issue known and is there a way to fix it? would any edit the `nextcloud.cfg` (max files or something ... ) file help for this?

Thanks again in advance.

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question at https://help.nextcloud.com. Thanks!
